src/state_types/models/notifications.rs

```diff
@@ -103,10 +103,12 @@ impl<Env: Environment + 'static> UpdateWithCtx<Ctx<Env>> for Notifications {
                                     // we have to get `now` somehow (environment or through a msg)
                                     // Alternatively, just set that when we add lib items
                                     .retain(|v| {
-                                        lib_item
-                                            .state
-                                            .last_vid_released
-                                            .map_or(false, |lvr| v.released > lvr)
+                                        v.released.map_or(false, |vr| {
```

Comment:
1) are you sure this is the same logic
2) released should be mandatory; if someone didn't provide it, then we'll default to 0 (1970)
3) this shows the benefit of rust - changing one small thing in the codebase will typecheck every usage; if this was JS or even TS we'd be fucked